assets/js/googlesitekit/datastore/ui/ui.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * `core/ui` data store: ui data
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+import isPlainObject from 'lodash/isPlainObject';
+
+const SET_UI_VALUES = 'SET_UI_VALUES';
+const SET_UI_VALUE = 'SET_UI_VALUE';
```

Comment:
These could be simplified to just `SET_VALUES` / `SET_VALUE`.

Reply:
Updated @felixarntz